src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -350,7 +350,7 @@ public void openSearch(PlayerProfile profile, String input, boolean addToHistory
         }
 
         ChestMenu menu = new ChestMenu(Slimefun.getLocalization().getMessage(p, "guide.search.inventory").replace("%item%", ChatUtils.crop(ChatColor.WHITE, input)));
-        String searchTerm = input.toLowerCase(Locale.ROOT);
+        String searchTerm = ChatColor.strip(input.toLowerCase(Locale.ROOT));
```

Comment:
```suggestion
        String searchTerm = ChatColor.stripColor(input.toLowerCase(Locale.ROOT));
```
I made this PR on my phone, forgot the exact method name lol